### DISCUSSION TITLE
Trying to randomise seed with load prompts from file for generation of 500 images.

### DISCUSSION THREAD

**Discussion by Miteas** [comfyanonymous/ComfyUI#2492] [Q&A] [answered]
I'm trying to create a large dataset of 500 images, with 500 prompts from a file. Using the node "Load prompts from file - Inspire"
 The issue I'm having is that the seed doesn't randomise for each prompt, instead the seed randomises once once the model starts, and uses the same seed for the 500 images.
 Is there a simple way around this?
 I'm open to using other nodes. 
 I'm aware this sounds like a strange use case, but it is an experiment.

**Comment by ltdrdata** [comment, marked as answer]
`Random Generator for List (Inspire)` is added. Update Inspire Pack and try this.

![image](https://github.com/comfyanonymous/ComfyUI/assets/128333288/10c14de1-2b7c-4a1e-b5dc-ceb2b0893049)

**Reply by Miteas** [reply]
You hero
